### PR TITLE
Alternative support for qualifiers

### PR DESCRIPTION
**Pull request by edeutsch**
Alternative to #325, based on discussion in #325

### CODE REVIEW

**Review by vdancik — CHANGES_REQUESTED** [review]
Can we be more specific, e.g. type => qualifier_type and value => qualifier_value

**Review by vdancik — APPROVED** [review]

File: TranslatorReasonerAPI.yaml [hunk 1]
```diff
@@ -784,10 +784,44 @@ components:
...
+          type: string
+          description: >-
+            The value associated with the type of the qualifier, drawn from
+            a set of controlled values by the type as specified in
+            the Biolink model (e.g. 'decreased' or 'increased' for the type
+            subject_direction, etc).
+          example: decreased
+          nullable: false
+      required:
+        - type
```
Comment (mbrush) [inline TranslatorReasonerAPI.yaml:823]: should this read 'qualifier_type_id'

File: TranslatorReasonerAPI.yaml [hunk 2]
```diff
@@ -784,10 +784,44 @@ components:
...
+            object_aspect, object_direction, etc).
+          example: subject_aspect
+          nullable: false
+        value:
+          type: string
+          description: >-
+            The value associated with the type of the qualifier, drawn from
+            a set of controlled values by the type as specified in
+            the Biolink model (e.g. 'decreased' or 'increased' for the type
+            subject_direction, etc).
```
Comment (mbrush) [inline TranslatorReasonerAPI.yaml:819]: Consider making the example here consistent with the example of a qualifier_type_id above (where you used 'subject_aspect').  So here, say (e.g. 'expression' or 'abundance' for the qualifier type 'subject_aspect', etc).

**Comment by edeutsch** [comment]
Closing this proposal in favor of #330

**Pull request closed without merging**